### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by 1GORDON**
# In this project, I achieved the following

- Used ES6 modules, with import and export
- Created an npm project with Webpack
- Wrote plain HTML markup with minimum styling
- Used Gitflow in this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
 # Leaderboard
-In this project I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I will develop a first working version of the app following a wireframe, but without styling - just focus on functionality
+
+In this project i acheived the following
+
```
Comment (eliasecasta) [inline README.md:4]: - [x] [OPTIONAL] I would highly recommend you to add a screenshot of your project to make your Readme look even better!

File: src/loadTable.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const tr = document.createElement('tr');
+    tr.className = 'table-row';
+    const td = document.createElement('td');
+    td.className = 'table-data';
+    td.innerHTML = `${data.name}: ${data.score}`;
+
+    tr.appendChild(td);
+    table.appendChild(tr);
+  });
+}
```
Comment (eliasecasta) [inline src/loadTable.js]: - [x] ES6 syntax must be used for this project, please change the function keyword for an arrow function structure to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @1GORDON, I am Elias, your reviewer for this PR

Well done so far with your project!

Everything works as expected and your repo looks professional, great job so far!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] ES6 syntax is required for this project, please use arrow functions.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] I would highly recommend you to add a screenshot of your project to make your Readme look even better!

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/loadTable.js (hunk 2 above)
Comment (1GORDON) [reply src/loadTable.js]: Thank you for the corrections

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @1GORDON, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**